lib/Doctrine/ORM/Proxy/Proxy.php

```diff
@@ -26,6 +26,8 @@
  *
  * @author Roman Borschel <roman@code-factory.org>
  * @since 2.0
+ *
+ * @deprecated This class is deprecated and will be removed in Doctrine 3.0, proxies will no longer implement it.
```

Comment:
Does it make sense to mention which interface should be used? So that people can already start preparing their stuff

Reply:
Not sure about this one, we would be referencing 3rd party code (ProxyManager) which is not required in Composer.